source/write/insert.txt

```diff
@@ -0,0 +1,221 @@
+  ``_id`` field value is unique.
+- You can let the driver automatically generate a unique ``ObjectId``
+  value for each document ``_id``. If you do not manually set an
+  ``_id`` value for a document, the driver populates the field
+  with an ``ObjectId``.
+
+Unless you can guarantee uniqueness, we recommend
+letting the driver automatically generate ``_id`` values.
+
+.. note:: Duplicate _id Errors
+
+   Setting duplicate ``_id`` values in a collection violates unique
+   index constraints, which causes the driver to return a ``WriteError`` from
+   the ``insertOne()`` method or a ``BulkWriteError`` from the
+   ``insertMany()`` method.
+
+To learn more about the ``_id`` field, see the
+:manual:`Unique Indexes </core/index-unique/>` guide in the
+{+mdb-server+} manual.
+
+To learn more about document structure and rules, see the
+:manual:`Documents </core/document>` guide in the {+mdb-server+} manual.
+
+Insert One Document
+-------------------
+
+To add a single document to a MongoDB collection, call the ``insertOne()``
+method and pass the document you want to add.
+
+The following example inserts a document into the ``restaurants``
+collection:
+
+.. literalinclude:: /includes/write/insert.kt
+   :start-after: start-insert-one
+   :end-before: end-insert-one
+   :language: kotlin
+   :copyable:
+   :dedent:
+
+Insert Multiple Documents
+-------------------------
+
+To add multiple documents to a MongoDB collection, user the ``insertMany()``
+method and pass a list of documents you want to add.
+
+The following example inserts a list of documents into the
+``restaurants`` collection:
+
+.. literalinclude:: /includes/write/insert.kt
+   :start-after: start-insert-many
+   :end-before: end-insert-many
+   :language: kotlin
+   :copyable:
+   :dedent:
+
+Modify Insert Behavior
+----------------------
+
+The ``insertOne()`` method optionally accepts a parameter
+that sets options to configure the insert operation.
+If you don't specify any options, the driver performs the insert
+operation with default settings.
+
+The following table describes the setter methods that you can use to
+configure an ``InsertOneOptions`` instance:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``bypassDocumentValidation()``
+     - | If set to ``true``, allows the driver to ignore
+         :manual:`document-level validation </core/schema-validation>`. 
+       | Defaults to ``false``.
+
+   * - ``comment()``
+     - | Sets a comment to attach to the operation. For more information, see the :manual:`insert command
+         fields </reference/command/insert/#command-fields>` guide in the
+         {+mdb-server+} manual for more information.
+
+You can set the preceding settings on the ``insertMany()`` method
+by configuring an ``InsertManyOptions`` instance. You can also use the
+``ordered()`` method setter method to specify the order in which the driver
```

Comment:
Q: Is ordered() only applicable to insertmanyoptions?

S: instead of having the ordered method be in the copy where people might overlook it, make a new table with just one entry and have the ordered() method in the table with a description.